contrib/podmanremoteimage/README.md

```diff
@@ -0,0 +1,26 @@
+podman-remote-images
+====================
+
+Overview
+--------
+
+This directory contains the containerfile for creating a container image which consist podman-remote binary
+for each platform (win/linux/mac). Plan is to hook it with quay github build trigger to generate this image
+for each PR merged (in any 4.x branch or tags).
+
+User can copy those binary on specific platform using following
+
+- For Windows binary
+```bash
+$ podman cp $(podman create --name remote-temp quay.io/containers/podman-remote-artifacts:latest):/windows/podman.exe . && podman rm remote-temp
+```
+
+- For Linux binary
+```bash
+$ podman cp $(podman create --name remote-temp quay.io/praveenkumar/podman-remote-artifacts:latest):/podman-remote-static . && podman rm remote-temp
```

Comment:
```suggestion
$ podman cp $(podman create --name remote-temp quay.io/containers/podman-remote-artifacts:latest):/podman-remote-static . && podman rm remote-temp
```

Reply:
Points to personal repo, are we expecting to create similar tag in `containers`

Reply:
I think similar comment goes for `darwin`

Reply:
@flouthoc yes I forgot to remove my personal repo :) . I think we should have similar repo/tag in `containers` org for consumption.